### ISSUE TITLE
Regularization layers

### ISSUE THREAD

**Issue by umangjpatel** [umangjpatel/dnet#11]
**Description**
Add regularization layers such as Inverted Dropout and Batch Norm

**Comment by umangjpatel** [comment]
Regularization APIs added :
1) Dropout
2) BatchNorm
